GitExtUtils/HexString.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text;
+
+namespace GitCommands
+{
+    public static class HexString
+    {
+        /// <summary>
+        /// Converts a byte array into its hex representation.
+        /// </summary>
+        public static string FromByteArray(byte[] data, bool lowerCase)
+        {
+            if (data is null)
+            {
+                return null;
+            }
+
+            if (data.Length == 0)
+            {
+                return string.Empty;
+            }
+
+            var formatString = lowerCase ? "x2" : "X2";
+            var builder = new StringBuilder(capacity: data.Length * 2);
+
+            foreach (var b in data)
+            {
+                builder.Append(b.ToString(formatString));
```

Comment:
> builder.Append(b.ToString(formatString));

`builder.AppendFormat($"0:{formatString}", b);`

Reply:
✔ done

Reply:
builder.AppendFormat("x2", b); => builder.AppendFormat("{0:x2}", b); ?

Reply:
damn it. Thank you - fixed it.